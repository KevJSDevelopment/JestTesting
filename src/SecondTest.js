import React from 'react'

const SecondTest = () => {
    return (
        <div>
            <h1>Second test</h1>
        </div>
    )
}

export default SecondTest