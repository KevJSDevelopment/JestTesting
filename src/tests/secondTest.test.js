import React from 'react'
import ReactDOM from 'react-dom'
import SecondTest from '../SecondTest'
import {render} from '@testing-library/react'

test('take a screenshot of the second test component', () => {
    const component = render(
        <SecondTest />
    )
    expect(component.container).toMatchSnapshot();
})

test('Check if component renders correctly', () => {
    const div = document.createElement('div')
    const component = render(
        <SecondTest />
    )
    expect(component.container).toMatchSnapshot();
})