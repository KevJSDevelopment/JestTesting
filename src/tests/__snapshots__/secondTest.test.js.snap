// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if component renders correctly 1`] = `
<div>
  <div>
    <h1>
      Second test
    </h1>
  </div>
</div>
`;

exports[`take a screenshot of the second test component 1`] = `
<div>
  <div>
    <h1>
      Second test
    </h1>
  </div>
</div>
`;

exports[`take a screenshot of the second test component 2`] = `
<div>
  <div>
    <h1>
      Second test
    </h1>
  </div>
</div>
`;
